controllers/public.controllers.js

```diff
@@ -0,0 +1,219 @@
+const ObjectId = require('mongoose').Types.ObjectId;
+const Basket = require('../models/basket.model');
+const User = require('../models/user.model');
+const Participant = require('../models/participant.model');
+
+const getParticipantsIds = async userId => {
+  const participants = await Participant.find({ user: userId }, '-_id basket');
+  return participants.map(el => el?.basket);
+};
```

Comment:
here an example with one "await", in code where u used two await u did the same but without extra await